src/hardware/gus.cpp

```diff
@@ -1771,12 +1812,12 @@ void init_gus_dosbox_settings(Section_prop& secprop)
 
 	auto* int_prop = secprop.Add_int("gusirq", when_idle, 5);
 	assert(int_prop);
-	int_prop->Set_values({"3", "5", "7", "9", "10", "11", "12"});
+	int_prop->Set_values({"2", "3", "5", "7", "11", "12", "15"});
 	int_prop->Set_help("The IRQ number of the Gravis UltraSound (5 by default).");
 
 	int_prop = secprop.Add_int("gusdma", when_idle, 3);
 	assert(int_prop);
-	int_prop->Set_values({"0", "1", "3", "5", "6", "7"});
+	int_prop->Set_values({"1", "3", "5", "6", "7"});
 	int_prop->Set_help("The DMA channel of the Gravis UltraSound (3 by default).");
```

Comment:
There must be some magical link between you and I @interloper98 because I was considering doing the same enhancements... 🥳 🧙🏻‍♂️🔮 🪄 😂   

..._OR_ you've also just seen the list of available options in the sound setup utility of the demo **Inside by CNCD** 😉 

Would fixing up `gusbase` be worthwhile too? Our current values:

<img width="913" alt="image" src="https://github.com/user-attachments/assets/2c60f9c0-cddc-4432-bd2d-46c051bea9bf">

Values from the sound setup utility:

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/bf6934bc-a52a-419b-9219-084dadf8b87c">

The IRQ and DMA settings are now in-line with the utility:

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/0daa4402-4f28-4949-9eb7-d30269fefae3">

<img width="1215" alt="image" src="https://github.com/user-attachments/assets/39a52e89-9d8e-4b76-8b4a-3fb33fca2db8">

Reply:
Updated the base addresses.